dashboard/app/controllers/passwords_controller.rb

```diff
@@ -16,6 +29,10 @@ def after_sending_reset_password_instructions_path_for(resource_name)
 
   private
 
+  def email_in_hoc_2016_signups?(email)
+    !PEGASUS_DB[:forms].where(email: email, kind: 'HocSignup2016').first.nil?
```

Comment:
Might be able to `!x.nil?` -> `x.present?` (Rails helper), or possibly `PEGASUS_DB[:forms].where(email: email, kind: 'HocSignup2016').any?`